_includes/sponsors.html

```diff
@@ -1,42 +1,109 @@
-{% if mobile == "true" %}
-	{% assign goldWidth = "50%" %}
-	{% assign mobileTag = " mobile" %}
-{% else %}
-	{% assign goldWidth = "80%" %}
-	{% assign mobileTag = "" %}
+{% if mobile == "true" %} {% assign goldWidth = "50%" %} {% assign mobileTag = "
```

Comment:
nit but indentation?

Reply:
Idk why but my formatting just does that